portal/app/api/v1/endpoint/farm_data_parser.py

```diff
@@ -45,7 +45,9 @@ def local_fan_info_parser(data):
         Parses json blob for local fan info page and returns the argument fields
         as individual dictionaries in json form
     """
-    farm_data = data[0].get("device_data")
+    with open('response.json') as json_file:
```

Comment:
Can you explain what this is doing. Where and when does this file get written? Why use a file and are you sure it'll exist?